test/integration/integration_test.py

```diff
@@ -0,0 +1,42 @@
+from re import compile, Pattern
+from threading import Timer
+from subprocess import PIPE, Popen, check_call
+
+command: str = "velocitas exec runtime-local"
+regex_dapr_app: Pattern[str] = compile(r".*You\'re up and running! Both Dapr and your app logs will appear here\.\n")
+regex_mqtt: Pattern[str] = compile(r".*mosquitto version \d+\.\d+\.\d+ running\n")
+regex_vdb: Pattern[str] = compile(r".*Listening on \d+\.\d+\.\d+\.\d+:\d+")
+regex_fc: Pattern[str] = compile(r".*dbcfeeder: Connected to data broker")
+regex_vs: Pattern[str]= compile(r".*DataBrokerFeederImpl: connected to data broker\.")
```

Comment:
you could merge these two if you just check for "connected to data broker."
Please mind the capital letter, so best would be to make it case insensitive.